src/components/course/repo/lessons.repo.js

```diff
@@ -0,0 +1,29 @@
+const { lessonModel } = require('./db/lesson');
```

Comment:
I do not like mapping files, folders and repos in the way of the old collections.